sdk/servicebus/Azure.Messaging.ServiceBus/tests/Amqp/AmqpSenderTests.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Messaging.ServiceBus.Amqp;
+using Azure.Messaging.ServiceBus.Core;
+using Azure.Messaging.ServiceBus.Diagnostics;
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Framing;
+using Moq;
+using Moq.Protected;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests.Amqp
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="AmqpSender" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
```

Comment:
nit: I think this isn't needed.

Reply:
We're not deriving from any base that would inherit it.   I think it should be here.

Reply:
It should only be necessary if a parameter is used - https://docs.nunit.org/articles/nunit/writing-tests/attributes/testfixture.html:
`Beginning with NUnit 2.5, the TestFixture attribute is optional for non-parameterized, non-generic fixtures. So long as the class contains at least one method marked with the Test, TestCase or TestCaseSource attribute, it will be treated as a test fixture.`

We are on version 3.13 - https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L247